### PR TITLE
Memory pair game by f3d0t

### PR DESCRIPTION
**Pull request by f3d0t**
### task name

[Demo](https://f3d0t.github.io/f3d0t/memory_pair_game/)
[Code base](https://github.com/f3d0t/f3d0t/tree/main/memory_pair_game)

The code is submitted in a dedicated feature branch.
Please, review.

### CODE REVIEW

**Comment by OleksiyRudenko** [comment]
@f3d0t it is really nice that this implementation uses a queue of items. This allows a player to click as many cards as they want, yet those analyzed in the order they were clicked in. 👍

File: submissions/f3d0t/memory_pair_game/app.js [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+	}
+	twoCardsArray.forEach((card) => {
+		setTimeout(() => {
+			card.classList.remove("flipped");
+		}, TURN_DELAY);
+	});
+};
+
+const checkIfWin = () => {
+    if (CARDS_CONTAINER.getElementsByClassName("solved").length == CARDS_CONTAINER.childNodes.length) {
```
Comment (zonzujiro) [inline submissions/f3d0t/memory_pair_game/app.js:78]: Do not rely on DOM itself, keep number of cards separately.

**Review by zonzujiro — CHANGES_REQUESTED** [review]
One comment. Other than that - LGTM

File: submissions/f3d0t/memory_pair_game/app.js (hunk 1 above)
Comment (f3d0t) [reply submissions/f3d0t/memory_pair_game/app.js]: Done, now it counts in separate value

**Comment by f3d0t** [comment]
Thank you 4 review =) Еhis is my least buggy application

**Pull request merged**